### PR TITLE
FORMS-9917 Submit successfull event is not working

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js [hunk 1]
```diff
@@ -141,4 +141,11 @@ describe("Form with Submit Button", () => {
         // To Fix: this is failing the test saying waiting for page load
         //cy.wait('@redirected');
     })
-})
+
+    it("Submit Action test without passing any custom submit event", () => {
+        cy.previewForm(basicSubmitPagePath);
+        cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
```
Comment (rismehta) [inline ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js:147]: Please write test case for submit success handler, this test already exists.

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

File: ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js (hunk 1 above)
Comment (dmaurya929) [reply ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js]: Thankyou message comes only after submit success handler is executed https://github.com/adobe/aem-core-forms-components/blob/f9d0f0d11ca156b48c7b58da7b45ab3174ecd123/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js#L39 
I have added event listener for`submitStart` event which is triggered after the submitSuccess event

**Review by sakshi-arora1 — APPROVED** [review]

**Comment by dmaurya929** [comment]
Added test to navigate to give URL if form is submitted successfully. This rule is working fine.
( I was also trying to write a test to navigate to an external url as well but facing issue with cypress. )

File: it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/actions/submit/basicsubmit/.content.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rismehta) [inline it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/actions/submit/basicsubmit/.content.xml:1]: Can you rename `basicsubmit` to `submitsuccessrule` ?

**Comment by sakshi-arora1** [comment]
Merging (as there are only 3 test failures, which are going to be fixed by my PR). Discussed tests have been added.
cc: @rismehta

**Comment by rismehta** [comment]
test failures are known and being worked upon by @sakshi-arora1. merging this

**Pull request merged**